sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v3/query.sql

```diff
@@ -37,7 +37,9 @@ WITH impressions AS (
 ),
 sanitized_queries AS (
   SELECT
-    *
+    TIMESTAMP_TRUNC(timestamp, SECOND) as timestamp,
+    LTRIM(LOWER(query)) as query,
+    * except (timestamp, query)
```

Comment:
```suggestion
    * except (timestamp, query, region, country)
```

I think it would be cleaner to remove these in this initial query rather than in the join.